### PR TITLE
(COV-1065): Setup Cron Job - Rt & Daily Cases - Cook County

### PR DESCRIPTION
**Pull request by nss10**
### New Features
* Add new job and cron job for the Rt & Daily Cases - Cook County (Generative Bayes Model)

### Deployment changes
* New jobs must be deployed using `gen3 job run covid19-bayes S3_BUCKET <S3_BUCKET>` and `gen3 job run covid19-bayes-cronjob S3_BUCKET <S3_BUCKET>`

### CODE REVIEW

File: kube/services/jobs/covid19-bayes-cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                    key: hostname
+              - name: S3_BUCKET
+                GEN3_S3_BUCKET
+              resources:
+                requests:
+                  cpu: 4
+                  memory: 20Gi
+                limits:
+                  cpu: 4
+                  memory: 20Gi # guess on upper bound
```
Comment (paulineribeyre) [inline kube/services/jobs/covid19-bayes-cronjob.yaml:53]: Let's not give it this much cpu/memory. We can ask @wangfan860 if she knows how much it needs, and if she doesn't, let's use a lower value (maybe [this](https://github.com/uc-cdis/cloud-automation/blob/573c3e83859cb54145b6704104e512f31bd68854/kube/services/jobs/covid19-notebook-etl-job.yaml#L47-L50)) and increase it if we need to

Comment (wangfan860) [reply kube/services/jobs/covid19-bayes-cronjob.yaml:53]: my test run was on a 30G, 16cores vm. I noticed that the actual job was using 4cores. for the memory, I would need to double check again to see how much it uses when it's peaked. @nss10 @paulineribeyre

File: kube/services/jobs/covid19-bayes-cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                    key: hostname
+              - name: S3_BUCKET
+                GEN3_S3_BUCKET
+              resources:
+                requests:
+                  cpu: 4
+                  memory: 12Gi
+                limits:
+                  cpu: 4
+                  memory: 12Gi # guess on upper bound
```
Comment (paulineribeyre) [inline kube/services/jobs/covid19-bayes-cronjob.yaml:53]: remove this comment

File: kube/services/jobs/covid19-bayes-job.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+              key: hostname
+        - name: S3_BUCKET
+          GEN3_S3_BUCKET
+        resources:
+          requests:
+            cpu: 4
+            memory: 12Gi
+          limits:
+            cpu: 4
+            memory: 12Gi # guess on upper bound
```
Comment (paulineribeyre) [inline kube/services/jobs/covid19-bayes-job.yaml:39]: here too

File: kube/services/jobs/covid19-bayes-cronjob.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# gen3 job run covid19-bayes-cronjob S3_BUCKET <S3_BUCKET>
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: covid19-bayes
+spec:
+  # Weekly Sunday, Wednesday 6:00PM Chicago time == Monday,Tuesday 12:00AM UTC
```
Comment (paulineribeyre) [inline kube/services/jobs/covid19-bayes-cronjob.yaml:7]:
```suggestion
  # Weekly Sunday, Wednesday 6:00PM Chicago time == Monday, Thursday 12:00AM UTC
```

**Review by wangfan860 — DISMISSED** [review]

**Review by paulineribeyre — APPROVED** [review]

**Comment by paulineribeyre** [comment]
The 2 failing test suites are currently broken. Everything else passed, so merging

**Pull request merged**